docs/src/pages/[platform]/components/input/InputPropControls.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  InputProps,
+  TextField,
+  SelectField,
+  SwitchField,
+  Flex,
+} from '@aws-amplify/ui-react';
+import * as React from 'react';
+
+export interface InputPropControlsProps extends InputProps {
```

Comment:
Just thinkin out loud, nothing to do with this PR, all this stuff should be automatically generated

Reply:
100%